data/json/construction.json

```
@@ -73,6 +73,60 @@
     "dark_craftable": true,
     "post_special": "done_trunk_plank"
   },
+  {
+    "type": "construction",
+    "id": "constr_paddy_field",
+    "group": "plow_a_paddy_field",
+    "category": "FARM_WOOD",
+    "required_skills": [ [ "survival", 1 ] ],
+    "time": "20 m",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "components": [
+      [ [ "clay_lump", 8 ] ],
+      [ [ "water", 4 ], [ "water_clean", 4 ] ],
+      [ [ "material_sand", 20 ], [ "material_soil", 2 ] ]
+    ],
+    "pre_terrain": "t_pit_shallow",
+    "post_terrain": "t_paddy",
+    "dark_craftable": true
+  },
+  {
+    "type": "construction",
+    "id": "constr_paddy_field_remove",
+    "group": "remove_a_paddy_field",
+    "category": "FARM_WOOD",
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "5 m",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "pre_terrain": "t_paddy",
+    "post_terrain": "t_dirt",
+    "dark_craftable": true
+  },
+  {
+    "type": "construction",
+    "id": "constr_clay_bog",
+    "group": "gather_clay_from_dirt",
+    "category": "OTHER",
+    "required_skills": [ [ "survival", 2 ] ],
+    "time": "30 s",
+    "components": [ [ [ "material_soil", 50 ] ], [ [ "water", 32 ], [ "water_clean", 32 ] ] ],
+    "pre_terrain": "t_pit_shallow",
+    "post_terrain": "t_clay_bog",
+    "dark_craftable": true
+  },
+  {
+    "type": "construction",
+    "id": "constr_clay_stirred",
+    "group": "gather_clay_from_dirt",
+    "category": "OTHER",
+    "required_skills": [ [ "survival", 2 ] ],
+    "time": "90 m",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "byproducts": [ { "item": "clay_lump", "charges": [ 6, 12 ] } ],
```

Consuming 90 minutes looks too much to get clay same as extracting.
So I cut the time to 2/3, and slightly increase the clay.

```suggestion
    "time": "60 m",
    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
    "byproducts": [ { "item": "clay_lump", "charges": [ 8, 16 ] } ],
```